src/script/interpreter.h

```diff
@@ -124,7 +124,8 @@ struct PrecomputedTransactionData
     uint256 hashPrevouts, hashSequence, hashOutputs;
     bool ready = false;
 
-    explicit PrecomputedTransactionData(const CTransaction& tx);
+    template<class T>
+    PrecomputedTransactionData(const T& tx);
```

Comment:
Why drop `explicit` here? I don't see changes to the call sites.

Reply:
you are right, I probably should not drop that.